CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+
+## [5.0.0] - 2025-01-07
+### Breaking Changes
```

Comment:
Please explicitly mention that using this SDK will result in a new smart contract address.